azurerm/internal/services/timeseriesinsights/parse/time_series_insights_environment.go

```diff
@@ -0,0 +1,33 @@
+package parse
+
+import (
+	"fmt"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+)
+
+type TimeSeriesInsightsEnvironmentId struct {
+	ResourceGroup string
+	Name          string
+}
+
+func TimeSeriesInsightsEnvironmentID(input string) (*TimeSeriesInsightsEnvironmentId, error) {
+	id, err := azure.ParseAzureResourceID(input)
+	if err != nil {
+		return nil, fmt.Errorf("[ERROR] Unable to parse Time Series Insights Environment ID %q: %+v", input, err)
```

Comment:
Could we please modify this error message  a bit to make it to look less like a log rather than an error message?
```suggestion
		return nil, fmt.Errorf("parsing Time Series Insights Environment ID %q: %+v", input, err)
```

Reply:
Good catch! Done!